bin/docker_utils.py

```diff
@@ -62,10 +62,12 @@ def create_repo(username, repo, private=False):
   response = hub_request("/repositories/", payload, method = 'POST')
   return (False, response, response.reason, response.text) if not response.ok else (response.ok,)
 
-def delete_repo(username, repo):
+def delete_repo(username, repo, force=False):
   uri = '/repositories/%s/%s' % (username, repo)
-  response = hub_request(uri, method = 'DELETE')
-  return (False, response, response.reason, response.text) if not response.ok else (response.ok,)
+  if get_tags('%s/%s'%(username,repo))[1] == [] or force:
```

Comment:
please change the order here and first check for `force` flag and then get tags e.g. something like
`if force or not get_tags('%s/%s'%(username,repo))[1]:`

this way we avoid calling get_tags if `force` is `True`